docs/CustomFunctions.md

```diff
@@ -0,0 +1,132 @@
+
+Please see the [specification](https://jmespath.org/specification.html#built-in-functions) for more details.
+
+## Mathmatical Functions
+
+### add
+
+```
+number add(number $left, number $right)
+```
+
+Returns the result of adding the left argument from the right.
+
+Examples:
+| Given                       | Expression       | Result |
+|-----------------------------|------------------|--------|
+| n/a                         | add(`10`, `10`)  | 20     |
+| {"left" : 40, "right" : 50} | add(left, right) | 90     |
+| {"left" : 0, "right" : 50}  | add(left, right) | 50     |
+
+### divide
+
+```
+number divide(number $left, number $right)
+```
+
+Returns the result of multiplying the left argument with the right.
+
+Examples:
+| Given                       | Expression          | Result |
+|-----------------------------|---------------------|--------|
+| n/a                         | divide(`10`, `10`)  | 1      |
+| {"left" : 40, "right" : 50} | divide(left, right) | 0.8    |
+| {"left" : 0, "right" : 50}  | divide(left, right) | 0      |
+
+### multiply
+
+```
+number multiply(number $left, number $right)
+```
+
+Returns the result of multiplying the left argument with the right.
+
+Examples:
+| Given                       | Expression            | Result |
+|-----------------------------|-----------------------|--------|
+| n/a                         | multiply(`10`, `10`)  | 100    |
+| {"left" : 40, "right" : 50} | multiply(left, right) | 2000   |
+| {"left" : 0, "right" : 50}  | multiply(left, right) | 0      |
+
+
+### pow
+
+```
+number pow(number $left, number $right)
+```
+
+Returns the result of raising the left argument to the power of the right.
+
+Examples:
+| Given                         | Expression       | Result                     |
+|-------------------------------|------------------|----------------------------|
+| n/a                           | pow(`10`, `10`)  | 10000000000                |
+| {"left" : 40, "right" : 50}   | pow(left, right) | mathmatic error : overflow |
+| {"left" : 0, "right" : 50}    | pow(left, right) | 0                          |
+| {"left" : 100, "right" : 0.5} | pow(left, right) | 10                         |
+
+### subtract
+
+```
+number subtract(number $left, number $right)
+```
+
+Returns the result of subtracting the left argument from the right.
+
+Examples:
+| Given                       | Expression            | Result |
+|-----------------------------|-----------------------|--------|
+| n/a                         | subtract(`10`, `10`)  | 0      |
+| {"left" : 40, "right" : 50} | subtract(left, right) | -10    |
+| {"left" : 0, "right" : 50}  | subtract(left, right) | -50    |
+
+## String Functions
+
+### fromUnixTimestamp
+
+```
+string fromUnixTimestamp(number $unixTimestamp)
+```
+
+Produces an [ISO 8061](https://en.wikipedia.org/wiki/ISO_8601) compliant time stamp from the given Unix timestamp.
```

Comment:
I gave feedback on your other PR about Unix timestamps being both seconds and milliseconds from reference. We should update the documentation when we decide on the change.

Reply:
I added an additional function `fromUnixTimestampMs` to support the milliseconds scenario. I've also updated the documentation to reflect this.